advanced_guides/ranking.md

```diff
@@ -8,12 +8,12 @@ MeiliSearch has default `criteria`.
 
 Here is the list of all the `criteria` that are executed in this specific order by default:
 
-- `Number of Typos` - The less typos there are beween the query words and the document words, the better is the document.
-- `Number of Words` - A document containing more of the query words will be more important than one that contains less.
-- `Words Proximity` - The closer the query words are in the document the better is the document.
-- `Attribute` - A document containing the query words in a more important attribute than another document is considered better.
-- `Position` - A document containing the query words at the start of an attribute is considered better than a document that contains them at the end.
-- `Exact` - A document containing the query words in their exact form, not only a prefix of them, is considered better.
+- `_typo`: The less typos there are beween the query words and the document words, the better is the document.
+- `_words` - A document containing more of the query words will be more important than one that contains less.
+- `_proximity` - The closer the query words are in the document the better is the document.
+- `_attribute` - A document containing the query words in a more important attribute than another document is considered better.
+- `_words_position` - A document containing the query words at the start of an attribute is considered better than a document that contains them at the end.
+- `_exact` - A document containing the query words in their exact form, not only a prefix of them, is considered better.
```

Comment:
I would like to call it `exactness` instead, I will come back to you if I change it.

Reply:
Allright! I like it better too.